modules/custom-tuning-specification.adoc

```diff
@@ -348,7 +348,7 @@ ifdef::rosa-hcp-tuning[]
 machine pools.
 endif::rosa-hcp-tuning[]
 
-This functionality takes advantage of `spec.providerID` node object values in the form of `<cloud-provider>://<cloud-provider-specific-id>` and writes the file `/var/lib/tuned/provider` with the value `<cloud-provider>` in NTO operand containers.  The content of this file is then used by TuneD to load `provider-<cloud-provider>` profile if such profile exists.
+This functionality takes advantage of `spec.providerID` node object values in the form of `<cloud-provider>://<cloud-provider-specific-id>` and writes the file `/var/lib/ocp-tuned/provider` with the value `<cloud-provider>` in NTO operand containers. The content of this file is then used by TuneD to load `provider-<cloud-provider>` profile if such profile exists.
```

Comment:
```suggestion
This functionality takes advantage of `spec.providerID` node object values in the form of `<cloud_provider>://<cloud-provider-specific-id>` and writes the file `/var/lib/ocp-tuned/provider` with the value `<cloud_provider>` in NTO operand containers. The content of this file is then used by TuneD to load `provider-<cloud_provider>` profile if such profile exists.
```

Reply:
Looking at the example, I think these are user replaced values that require `_`.  I wasn't sure if the <cloud_provider_specific_id> needed to be changed or not as well.